File: fizzbuzz.js
for(var i = 1; i <= 100; i++){
	if(i % 3 == 0 && i % 5 == 0)
		print("fizzBuzz");
	else if(i % 3 == 0)
		print("fizz");
	else if(i % 5 == 0)
		print("buzz");
	else
		print(i);
}

File: fizzBuzz.jl
function fizzbuzz(x)
  string = ""
  if x % 3 == 0
    string = string * "fizz"
  end
  if x % 5 == 0 
    string = string * "buzz"
  end

  if string == ""
    string = x
  end
  println(string)
end

[fizzbuzz(i) for i = 1:100]
